### PR TITLE
Announce App Tools v0.7.1

### PR DESCRIPTION
**Pull request by baleboy**
Blog post announcing the new App Tools release.

### CODE REVIEW

**Comment by baleboy** [comment]
@bspencer @r0b5t4

**Comment by baleboy** [comment]
I added a note about contributions

File: public/blog/_data.json [hunk 1]
```diff
@@ -1,4 +1,9 @@
 {
+  "crosswalk-app-tools-07": {
+    "title": "Crosswalk App Tools, the easiest way to package Crosswalk Applications",
```
Comment (r0b5t4) [inline public/blog/_data.json]: What about "The fastest way to package Web Applications" (but yours is good too)

Comment (baleboy) [inline public/blog/_data.json]: changed

**Comment by r0b5t4** [comment]
BTW i had to do 0.7.2 yesterday because one file was not packaged (only relevant when using app-tools as a library e.g. grunt integration, not command-line)

**Pull request merged**